src/Microsoft.AspNet.Authentication/AuthenticationTicket.cs

```diff
@@ -45,5 +45,10 @@ public AuthenticationTicket(ClaimsPrincipal principal, AuthenticationProperties
         /// Additional state values for the authentication session.
         /// </summary>
         public AuthenticationProperties Properties { get; private set; }
+
+        /// <summary>
+        /// Holds error information caused by authentication.
+        /// </summary>
+        public ErrorContext Error { get; set; }
```

Comment:
Hum, not sure about that. `AuthenticationTicket` is supposed to be fully serializable but this type has a name that may suggest it's not really meant to be persisted :smile: 

Is it even needed, since `AuthenticateResult` has an `Exception` property?